miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -36,15 +40,81 @@ interface MiniAppFileChooser {
     ): Boolean
 }
 
+/**
+ * A class to provide the interfaces for getting and requesting camera permission.
+ */
+interface MiniAppCameraPermissionDispatcher {
+    /**
+     * Get camera permission from host app.
+     * You can also throw an [Exception] from this method.
+     */
+    fun getCameraPermission(
+        onSuccess: (isGranted: Boolean) -> Unit,
+        onError: (message: String) -> Unit
```

Comment:
Does `getCameraPermission` really need to have success and error callbacks?
It seems like it would always return either a `true` or `false` value, or is there some case where it would return an error?

Reply:
@corycaywood you are right!! we can just use one `callback(isGranted: Boolean)` and depending on `isGranted` we can decide the next step